makefile

```diff
@@ -18,10 +18,6 @@ install:
 	go mod tidy
 	go mod vendor
 
-install-generator:
-	go get -u aqwari.net/xml/cmd/xsdgen
-	go install aqwari.net/xml/cmd/xsdgen
-
```

Comment:
What are the new install steps?

Reply:
good catch. i need to update that.

Reply:
where are we going to store the generator? it's a modified version of a library different from the one we use now, which gives us the ability to use different templates for RTP, FedNow, and standard ISO20022 (or anything else we want) along with the ability to template custom overrides. i don't want it to be in rtp20022 because of its universal appeal.

Reply:
We should offer a PR (or multiple if needed) to [the original repository](https://github.com/GoComply/xsd2go) to address the issues.

Reply:
Can you share the diff? We should at least host it on moov-io/xsd2go if the new version is incompatible.

Reply:
https://github.com/moov-io/xsd2go/pull/1